Makefile

```diff
@@ -101,8 +101,9 @@ create-test-db-file: run-migrations
 
 gen-apidoc:
 	rm -fr $(PYDIR)/staticfiles/
+	rm -fr $(APIDOC)
 	apidoc -i $(PYDIR) -o $(APIDOC)
-	cp docs/source/specs/openapi.yml $(APIDOC)/
```

Comment:
Need a `mkdir $(APIDOC)`?

I hit this 
```
(koku_master-2goqrU2u) ➜  koku_master git:(646-serve-openapi) make gen-apidoc
rm -fr koku/staticfiles/
rm -fr apidoc
apidoc -i koku -o apidoc
2019-02-19 14:08:08,155 - ERROR - Failed to parse sources
cp docs/source/specs/openapi.json apidoc/
cp: directory apidoc does not exist
make: *** [gen-apidoc] Error 1
```

Even after this it's still failing for me.  Am I doing something wrong?
```
(koku_master-2goqrU2u) ➜  koku_master git:(646-serve-openapi) ✗ make gen-apidoc
rm -fr koku/staticfiles/
rm -fr apidoc
mkdir apidoc
apidoc -i koku -o apidoc
2019-02-19 14:13:21,719 - ERROR - Failed to parse sources
cp docs/source/specs/openapi.json apidoc/
(koku_master-2goqrU2u) ➜  koku_master git:(646-serve-openapi) ✗
```

Reply:
Also adding ApiDoc to Pipfile would be great

Reply:
Strange I get the following:
```
$ make gen-apidoc
rm -fr koku/staticfiles/
rm -fr apidoc
apidoc -i koku -o apidoc
info: Done.
cp docs/source/specs/openapi.json apidoc/
```

[apidoc](http://apidocjs.com/) is NodeJS so can't add it to the Pipfile.

Reply:
🤦‍♂️  ok that explains it :)

Reply:
Not your fault its only kinda documented through the code install. I'll open a bug to doc that part.

Reply:
https://github.com/project-koku/koku/issues/651